Add database routing and models representation in Django admin

lkulbacki opened the pull request:
Register model's admin and create database routers as project uses three of them.

lkulbacki committed:
Configure ModelAdmin for offers and products and register with admin

lkulbacki committed:
Update README.md

umgelurgel commented on offers/admin.py:
```diff
@@ -1,3 +1,39 @@
+    in offers app context.
+    '''
+    # A handy constant for the name of the alternate database.
+    using = 'offers'
```
Ciekawe, dobre rozwiazanie. Te class methods to wszystkie ktore wymagaja sprecyzowania z jakiej DB pochodzi model?

lkulbacki replied:
Tak, to jest według dokumentacji kompletna definicja class methods.
Lista wymaganych metod jest krótsza, jeśli definicja klasy dziedziczy po InlineModelAdmin - ale nie zagłębiałem się w różnice z ModelAdmin wykorzystywanym tutaj.

umgelurgel commented on products/admin.py:
```diff
@@ -1,3 +1,41 @@
+from . import models as products_models
+
+
+class MultiDBModelAdmin(admin.ModelAdmin):
```
It'd be worth it to declare this class in another file along with the required subclasses so that you can simply import it here and in `products/offers` instead of fully redefining it; You'd only need to override the `using` value as needed, e.g.:

products/admin_base.py
```
class MultiDBModelAdmin(admin.ModelAdmin):
    using = None
    [...]

class OffersDBModelAdmin(admin.ModelAdmin):
    using = 'offers'
```

offers/admin.py:
```
from products.admin_base import OffersDBModelAdmin

admin.site.register(Offer, OffersDBModelAdmin)
```

lkulbacki replied:
Good point. I have created an abstract class in this commit: https://github.com/lkulbacki/datality-admin-backend/commit/f154fe0733d5cf38cdf91a4a5ee131f4f82f2ebd

umgelurgel reviewed (changes requested):
Looks good! Left a minor comment.

lkulbacki committed:
Create multidb base class definition

This is to simplify admin database router definition in applications
using legacy/external databases

umgelurgel reviewed (approved):
LGTM! :shipit:
